beets/autotag/mb.py

```diff
@@ -285,6 +289,27 @@ def album_info(release):
     artist_name, artist_sort_name, artist_credit_name = \
         _flatten_artist_credit(release['artist-credit'])
 
+    ntracks = sum(len(m['track-list']) for m in release['medium-list'])
+
+    # The MusicBrainz API omits 'artist-relation-list' and 'work-relation-list'
+    # when the release has more than 500 tracks. So we use browse_recordings
+    # on chunks of tracks to recover the same information in this case.
+    if ntracks > BROWSE_MAXTRACKS:
+        log.info(u'Album ' + str(release['id']) + u' has too many tracks')
+        log.info(u'Fetching recordings in batches of ' + str(BROWSE_CHUNKSIZE))
+        recording_list = []
+        for i in range(0, ntracks, BROWSE_CHUNKSIZE):
+            recording_list.extend(musicbrainzngs.browse_recordings(
+                    release=release['id'], limit=BROWSE_CHUNKSIZE,
+                    includes=BROWSE_INCLUDES,
+                    offset=BROWSE_CHUNKSIZE * i)['recording-list'])
```

Comment:
Just checking… shouldn't this just be `i`, not `BROWSE_CHUNKSIZE * i`? (Have you tried this out to make sure it's requesting the right tracks?)

Reply:
That's correct!